main.go

```diff
@@ -91,7 +93,8 @@ func init() {
 	flag.Int64Var(&schemaSampleSize, "schema-sample-size", int64(100000), "schema-sample-size: the number of rows to use for inferring schema (only for DynamoDB)")
 	flag.BoolVar(&verbose, "v", false, "verbose: print additional output")
 	flag.BoolVar(&schemaOnly, "schema-only", false, "schema-only: mode for schema only conversion to Spanner")
-
+	flag.BoolVar(&dataOnly, "data-only", false, "data-only: mode for data only conversion to Spanner")
```

Comment:
How about:

"data-only mode: in this mode we skip schema conversion and just do data conversion (use the session flag to specify the session file for schema and data mapping)"

and maybe something similar for schema-only:

"schema-only: in this mode we do schema conversion, but skip data conversion"

Reply:
Yup, sounds good. Updated.